src/status_im/translations/en.cljs

```diff
@@ -375,7 +375,7 @@
    :transactions-filter-type              "Type"
    :transactions-filter-select-all        "Select all"
    ;; Wallet Send
-   :wallet-send-transaction               "Send Transaction"
+   :wallet-send-transaction               "Send a Transaction"
```

Comment:
is it correct with article?

Reply:
There's disagreement between ios and android screens at the moment.

Reply:
designers should fix mockups